assets/components/organisms/footer-light/footer-light.twig

```diff
@@ -0,0 +1,29 @@
+<footer class="footer-light">
+  <div class="row">
+    <div class="col-xl-11 mx-xl-auto">
+      <div class="row">
+        <div class="col-6 mx-auto mx-md-0 mb-4 col-md-3 col-lg-2">
```

Comment:
Maybe a bit much helpers, as we try to keep markup as clean as possible for developers to implement this easier. 

(it's just a thought, work perfectly)